calico/README.md

```diff
@@ -15,11 +15,11 @@ Run the container
 ```console
 # Run as calico-node
 $ docker run -d --read-only --name=calico \
-    quay.io/cybozu/calico:3.14.1 start_runit
+    quay.io/cybozu/calico:3.16.1 start_runit
```

Comment:
3.16.1 does not exist.
```suggestion
    quay.io/cybozu/calico:3.16 start_runit
```

Reply:
I think [calico v3.6.1](https://github.com/projectcalico/calico/releases/tag/v3.16.1) exits. Did I miss something?

Reply:
@ysksuzuki It's the container image tag.